index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello world</title>
+</head>
+<body>
+    <h1>Hello World!</h1>
+    <h1 id="title">freeCodeCamp Survey Form</h1>

```

Comment:
- [ ] To follow HTML best practices, it's recommended to use only one h1 tag per page. Therefore, it's best to avoid using multiple h1 tags and instead use other heading tags, such as h2, h3, and so on, to structure the content on your page. This not only makes your code more semantically correct but also helps improve accessibility.